src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -778,10 +768,6 @@ public static ICC_Profile getInstance(byte[] data) {
 
         Profile p = null;
 
-        if (ProfileDeferralMgr.deferring) {
-            ProfileDeferralMgr.activateProfiles();
```

Comment:
Why do we need to load standard profiles if the user wants to load some other profile from the data array?